.docs/content/2.guide/1.how-to/how-to-use-htc-mock.md

```diff
@@ -0,0 +1,72 @@
+# How to use HTC Mock?
+
+HTC Mock is a test tool for ArmoniK. It is used to generate tasks that will be runned by ArmoniK. It provides a lot of options to personalize the tasks.
+
+HTC Mock is used in CI to test ArmoniK automatically but it can also be used in development to generate realistic tasks.
+
+::alert{type="info"}
+To populate the database, you can also use some scripts to populate database. See [Populate database?](../populate-database.md)
+::
+
+## Use it
+
+::alert{type="warning"}
+HTC Mock is a development or test tool. It is not intended to be used in production.
+::
+
+To use HTC Mock, you need to swap out the ArmoniK worker in `/infrastructure/quick-deploy/localhost/armonik/parameters.tfvars` with the HTC Mock worker.
+
+```diff [parameters.tfvars]
+    worker = [
+      {
+        name              = "worker"
+-       image             = "dockerhubaneo/armonik_worker_dll"
++       image             = "dockerhubaneo/armonik_core_htcmock_test_worker"
+        tag               = "0.12.1"
+    ]
+```
+
+Then, you can update the tag version. You can verify the [latest of Core on GitHub](https://github.com/aneoconsulting/ArmoniK.Core/release/latest) in order to use it.
+
+:warning: `dockerhubaneo/armonik_control` `dockerhubaneo/armonik_pollingagent` and `dockerhubaneo/armonik_core_htcmock_test_worker` must have the **exact** same tag version. It's a necessity to be sure that they can talk each other using the same API.
```

Comment:
:warning: `dockerhubaneo/armonik_control` `dockerhubaneo/armonik_pollingagent` and `dockerhubaneo/armonik_core_htcmock_test_worker` must have the **exact** same tag version. It is necessary so they can talk to each other using the same API.